syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -566,6 +594,24 @@ def chunk(tensor_shares, *args, **kwargs):
 
         module.chunk = chunk
 
+        @overloaded.function
+        def roll(tensor_shares, shifts, **kwargs):
```

Comment:
Can you add a docstring to explain the behaviour of this function for people who don't know the original torch.roll?